source/puddlestuff/tagsources/discogs.py

```diff
@@ -290,12 +297,13 @@ def search(artist=None, album=None):
         keywords = album
     return keyword_search(keywords)
 
+
 def urlopen(url):
     url = iri_to_uri(url)
     request = urllib2.Request(url)
     request.add_header('Accept-Encoding', 'gzip')
-    if puddlestuff.tagsources.user_agent:
```

Comment:
Leave this as, the user agent could get changed (bad form, I know, but I wrote the code 5 years ago). 

If you import the user agent only once, it won't get updated if the user updates their preferences.